apps/website/astro.config.ts

```diff
@@ -27,24 +29,27 @@ export default defineConfig({
       ],
     }),
     react(),
+    astroExpressiveCode({
+      themes: [nightOwl],
+      frames: { showCopyToClipboardButton: false },
+      cascadeLayer: 'thirdparty.ec',
+      minSyntaxHighlightingColorContrast: 4.5,
+      emitExternalStylesheet: true,
+      styleOverrides: {
+        borderColor: 'var(--color-line-2)',
+      },
+    }),
     mdx(),
     sitemap(),
   ],
   markdown: {
-    syntaxHighlight: 'prism',
     rehypePlugins: [rehypeSlug, [rehypeAutolinkHeadings, { behavior: 'wrap' }], rehypeToc],
   },
   server: { port: 1700 },
   scopedStyleStrategy: 'where',
   vite: {
     ssr: {
-      noExternal: [
-        '@fontsource/noto-sans',
-        '@tippyjs/react',
-        '@itwin/itwinui-react',
-        !isDev && '@itwin/itwinui-icons-react',
```

Comment:
Just curious, why is this not needed anymore?

Reply:
i believe this was needed before we fixed ESM support in icons